problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,84 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob Use set-builder notation to describe the following sets.
+		\begin{enumerate}
+			\item The set of vectors in $\R^{2}$ whose coordinates are rational numbers.
+
+			\item The set of vectors in $\R^{2}$ whose coordinates are irrational
+				numbers.
+
+			\item Let $P(\vec x) = \{\vec x, -\vec x\}$. The set $\{P(\vec e_{1}), P(
+				\vec e_{2})\}$.
+		\end{enumerate}
+
 		% Challenge (3 questions)
+		\prob % Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your
+		answers.
+		\begin{enumerate}
+			\item $\mathcal{Y}$, the y-axis in $\R^{3}$.
+				\begin{enumerate}
+					\item $\mathcal{Y}$ is a finite set.
+
+					\item Let
+						$\mathcal{A} = \{\vec a \in \R^{3} : \vec a = \beta \vec v
+						\text{ for some }\vec v \in \mathcal{Y}, \beta \in \R \}$, then $\mathcal{A}
+						\subseteq \mathcal{Y}$.
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, $\vec v \neq \vec 0$。
```

Comment:
Use `we have` after the comma so it doesn't look  like a bunch of math mushed together.